b2t.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/sh 
+
+# Translate BlockchainTests into t8n format.
+#
+# Note: t8n requires that hex values not have leading 0s, so you'll need to
+# trim the manually (for now) :(.
+
+set -euf
```

Comment:
*nitpick* 

expanding these flags to their longform is good self-documenting script code.

Reply:
Unfortunately i) set doesn't have longform flags and ii) I didn't document _why_ I put those flags there so now I've forgotten! :(

Reply:
https://www.gnu.org/software/bash/manual/html_node/The-Set-Builtin.html

See the `-o`:

```
set -o errexit -o nounset -o functrace
```

Though I'm not really sure this is objectively better now that I wrote it out....

Reply:
Might specifically be `bash` syntax too. Other shells might only support the short options.